### DISCUSSION TITLE
how to generate only cert and intermeidate file without root certificate authority

### DISCUSSION THREAD

**Discussion by simo222** [acmesh-official/acme.sh#5782] [General]
hi, 

the acme.sh scirpt generates a ca file which contains the root and intermediate.   well I don't need the root . 
 
is there an option to generate ? 

a)  only the certificate and intermediate without root. 

or 

b) only the intermediates  wihtout root  ca. 

according to rfc 

https://www.rfc-editor.org/rfc/rfc5246#appendix-F.1.1  
If the server is authenticated, its certificate message must provide a valid certificate chain leading to an acceptable certificate authority.

acme.sh generates a ca file however this one has a  root inside . 

```
server {
    listen 443 ssl;
    server_name myserver.domain.com ;

    ssl_certificate /etc/nginx/ssl/ myserver.domain.comm.crt;
    ssl_certificate_key /etc/nginx/ssl/myserver.domain.com.key;
    ssl_trusted_certificate /etc/nginx/ssl/ myserver.domain.com.ca.crt;

 location / {
          root /usr/share/nginx/html;
          index index.html;
      }
   } 
```
   
   for **/etc/nginx/ssl/ myserver.domain.com.ca.crt**  I want only the intermediates to be here. no root. 
   
is this possible  ?
just want to follow rfc.

**Comment by neilpang** [comment]
no,  the ca file only contains the  intermediate certs,  not including the root certs.
